services/api-server/src/simcore_service_api_server/services_rpc/wb_api_server.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import dataclass
+from functools import partial
+
+from fastapi import FastAPI
+from fastapi_pagination import Page, create_page
+from servicelib.rabbitmq._client_rpc import RabbitMQRPCClient
+from servicelib.rabbitmq.rpc_interfaces.webserver.licenses.licensed_items import (
+    get_licensed_items as _get_licensed_items,
+)
+
+from ..exceptions.service_errors_utils import service_exception_mapper
+from ..models.pagination import PaginationParams
+from ..models.schemas.model_adapter import LicensedItemGet
+
+_exception_mapper = partial(service_exception_mapper, service_name="WebApiServer")
+
+
+@dataclass
+class WbApiRpcClient:
+    _rabbitmq_rpc_client: RabbitMQRPCClient
```

Comment:
MINOR: both the class and the type incorporate that this is a rabbitMQ rpc client.  I would just call it `_client`

Reply:
Good point. Fixed